hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/UtilizationEndpoint.java

```diff
@@ -98,4 +111,44 @@ public Response getFileCounts(
     }
     return Response.ok(resultSet).build();
   }
+
+  /**
+   * Return the container size counts from Recon DB.
+   *
+   * @return {@link Response}
+   */
+  @GET
+  @Path("/containerCount")
+  public Response getContainerCounts(
+      @QueryParam(RECON_QUERY_CONTAINER_SIZE)
+          long containerSize) {
+    DSLContext dslContext = utilizationSchemaDefinition.getDSLContext();
+    Long containerSizeUpperBound =
+        ReconUtils.getContainerSizeUpperBound(containerSize);
+    List<ContainerCountBySize> resultSet;
+    try {
+      if (containerSize > 0) {
+        // Get the current count from database and update
+        Record1<Long> recordToFind =
+            dslContext.newRecord(
+                    CONTAINER_COUNT_BY_SIZE.CONTAINER_SIZE)
+                .value1(containerSizeUpperBound);
+        ContainerCountBySize record =
+            containerCountBySizeDao.findById(recordToFind.value1());
+        resultSet = record != null ?
+            Collections.singletonList(record) : Collections.emptyList();
+      } else {
+        // fetch all records having values greater than zero
```

Comment:
why don't we fetch all the records, even if a container size has 0 occurance?

Reply:
I don't think we need to consider containers with a size of zero, as they likely don't exist. Additionally, including the count of zero-sized containers in the graph may not be meaningful.